src/CommandQueue.ts

```diff
@@ -65,6 +80,9 @@ export class CommandQueue {
             } else {
                 throw new Error('IncorrectParametersException');
             }
+        },
+        addPlugin: (plugin: Plugin) => {
+            this.orchestration.addPlugin(plugin);
```

Comment:
Do we want type checks here to make sure the passed value is a plugin? (not every consumer will be using typescript)